python/pyspark/version.py

```diff
@@ -16,4 +16,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__: str = "2815!3.3.2+affirm.1"
+__version__: str = "2815!3.3.2+affirm.2"
```

Comment:
Nice saw that you added the changes here too:

https://docs.google.com/document/d/1sJ9Ou1uwqImuvwYORlPh0NymVZPjgald5gI-7Zc3g9c/edit#heading=h.bbobyguylhk9